source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Continuous Integration with CircleCI
```

Comment:
The two guides should also cross-link to each other in a brief sentence near the top.